aspnetcore/tutorials/nano-server.md

```diff
@@ -160,12 +160,11 @@ New-IISConfigCollectionElement $modules -ConfigAttribute @{"name"="AspNetCoreMod
 
 ## Installing .NET Core Framework
 
-If you published a Framework-dependent (portable) app, .NET Core must be installed on the target machine. Execute the following PowerShell script in a remote PowerShell session to install the .NET Framework on your Nano Server.
+If your app is a [framework-dependent deployment (FDD)](/dotnet/core/deploying/#framework-dependent-deployments-fdd), .NET Core must be installed on the server. Use the [dotnet-install.ps1 PowerShell script](https://raw.githubusercontent.com/dotnet/cli/release/2.0.0/scripts/obtain/dotnet-install.ps1) in a remote PowerShell session to install .NET Core on your Nano Server. Pass the CLI version with the `-Version` switch:
```

Comment:
"https://raw.githubusercontent.com/dotnet/cli/release/2.0.0/scripts/obtain/dotnet-install.ps1" should be: "https://dot.net/v1/dotnet-install.ps1"

Reply:
Thanks @johnbeisner. I'll update this as soon as I get back to the house.

Do u have any new info on https://github.com/dotnet/cli/issues/6747? ... That will block us here unless there is a workaround that I can describe.